tests/runtime/smw_strings/wasm.js

```diff
@@ -22,6 +22,7 @@ export function test_imports() {
   assertEq(b, "🚀");
   assertEq(c, "hello");
 }
+export { test_imports as "test-imports" }
```

Comment:
I think we'll want to keep this with JS the way it was before since it's the idioms-of-the-language? (e.g. we won't ask for `#[export_name]` on all the Rust things)

Reply:
Ah, I misunderstood how these tests work. I've now found the place in gen-spidermonkey to add the `to_snake_case()`. With that fixed, I've now reverted these test changes as they're no longer needed.